lib/changeNIDMtoxSPM.m

```diff
@@ -63,7 +83,10 @@
     %M
     
     %Locate the excursion set maps and their indices in the graph.
-    [excursionSetMaps, excurIndices] = searchforType('nidm_ExcursionSetMap', graph);
+    [excursionSetMaps, ~] = searchforType('nidm_ExcursionSetMap', graph);
```

Comment:
Ideally, rather than calling `searchforType` which means that we are going through the whole graph again, we would save the type of each object when first traversing the graph (in `addExcursionPointers`). This would lead to the same results but be more time efficient.